include/bluetooth/iso.h

```diff
@@ -456,12 +473,13 @@ struct bt_iso_server {
 	 *  This callback is called whenever a new incoming connection requires
 	 *  authorization.
 	 *
-	 *  @param acl The ACL connection that is requesting authorization
+	 *  @param info The ISO accept information structure
 	 *  @param chan Pointer to receive the allocated channel
 	 *
 	 *  @return 0 in case of success or negative value in case of error.
 	 */
-	int (*accept)(struct bt_conn *acl, struct bt_iso_chan **chan);
+	int (*accept)(const struct bt_iso_accept_info *info,
+		      struct bt_iso_chan **chan);
```

Comment:
`struct bt_iso_chan` is forward declared in the beginning of this file at line 77, which makes me think of it as an opaque structure, but it isnt.

Reply:
Can be removed, but will do so in another PR.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/38858